packages/client/components/Menu.tsx

```diff
@@ -157,10 +157,8 @@ const Menu = forwardRef((props: Props, ref: any) => {
   const handleKeyDown = useCallback(
     (e: React.KeyboardEvent) => {
       if (e.key === 'ArrowDown') {
-        e.preventDefault()
         setSafeIdx(activeIdx === null ? null : activeIdx + 1)
       } else if (e.key === 'ArrowUp') {
-        e.preventDefault()
```

Comment:
@BartoszJarocki After some testing, we concluded that the default action to the event `onKeyDown` allows the parent element to automatically scroll to any focusable children elements. Since `e.preventDefault()` was nuked on `ArrowDown` and `ArrowUp`, the default behavior described in the block above was reinstated. This allows for smooth scrolling down and up with the keyboard arrow keys.